core/src/main/java/cucumber/api/Result.java

```diff
@@ -15,8 +25,8 @@
         PASSED,
         SKIPPED,
         PENDING,
-        UNDEFINED,
         AMBIGUOUS,
+        UNDEFINED,
```

Comment:
`Ambiguous` and `Failed` both unconditionally make the exit code non-zero, but `Undefined` does not, so `Undefined` must be less severe than `Ambiguous`.

Reply:
You're right.